ledger/acctupdates.go

```diff
@@ -1846,6 +1885,7 @@ func (au *accountUpdates) vacuumDatabase(ctx context.Context) (err error) {
 	// rowid are flushed.
 	au.baseAccounts.prune(0)
 	au.baseResources.prune(0)
+	au.baseKV.prune(0)
```

Comment:
What can we do to get coverage of this? It seems important.

Reply:
My brief research indicates that this method is only called during specific db migrations triggered in `trackerdb.go`'s `runMigration` function. ie only when the ledger is being reloaded during an upgrade of the db schema version from 2 to 3. Seems difficult to test considering we're on version 8 and there aren't any tests I can find for the 2->3 migration.

Reply:
Ok, I was thinking this happened periodically.